app/renderer/components/Inspector/Actions.js

```diff
@@ -104,7 +105,14 @@ export default class Actions extends Component {
         {
           !_.isEmpty(pendingAction.action.args) && _.map(pendingAction.action.args, ([argName, argType], index) => <Row key={index} gutter={16}>
             <Col span={24} className={InspectorStyles['arg-container']}>
-              {argType === NUMBER && <Input type="number" value={pendingAction.args[index]} addonBefore={t(argName)} onChange={(e) => setActionArg(index, _.toNumber(e.target.value))} />}
+              {
+                argType === NUMBER && <Input
+                  type={INPUT.NUMBER}
+                  value={pendingAction.args[index]}
+                  addonBefore={t(argName)}
+                  onChange={(e) => setActionArg(index, _.toNumber(e.target.value))}
+                />
+              }
               {argType === STRING && <Input addonBefore={t(argName)} onChange={(e) => setActionArg(index, e.target.value)}/>}
```

Comment:
I was thinking about adding a linter rule to enforce each property to be on a new line. What do you think @KazuCocoa ?

Reply:
I also thought if we could coordinate syntax by lint